recipes/r-captioner/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = '2.2.3' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-captioner
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: captioner_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/captioner_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/captioner/captioner_{{ version }}.tar.gz
+  sha256: 72f92d757a47bae883cc95eb60a115c141aaf5eddcc15bc8068ea0faea15e775
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('captioner')"  # [not win]
+    - "\"%R%\" -e \"library('captioner')\""  # [win]
+
+about:
+  home: https://github.com/adletaw/captioner
+  license: Custom
```

Comment:
User "Other" instead of `Custom`. (We don't really have a standard for this but I think most of the feedstocks are using "Other").